### PR TITLE
Add some roadmaps for .net devs

### PR DESCRIPTION
**Pull request by Piatkosia**

### CODE REVIEW

File: Learning Paths/dot_net_backend_developer.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ In progress...
 
 🎓 .NET Backend Developer - Courses / Certification
 -----
-In progress...
+[Develop Web Apps with ASP.NET Core](https://learn.microsoft.com/en-us/collections/8mq4im72x6xprg)
```
Comment (bosterptr) [inline Learning Paths/dot_net_backend_developer.md:45]: Please change it from [Develop Web Apps with ASP.NET Core](https://learn.microsoft.com/en-us/collections/8mq4im72x6xprg)
to
Develop Web Apps with ASP.NET Core - [link](https://learn.microsoft.com/en-us/collections/8mq4im72x6xprg)

File: .gitignore
Comment (bosterptr) [inline .gitignore:1]: gitignore is unlikely to be needed, as Visual Studio is unlikely to add anything automatically to the repository with just md files. If I am wrong, please correct me

Comment (Piatkosia) [reply .gitignore:1]: It prevent to added unnecessary files by mistake. I can remove that if needed.

**Review by notthehiddenwiki — APPROVED** [review]
Approved

**Pull request merged**